src/engine/cluster/pilot_quantum_executor.py

```diff
@@ -0,0 +1,100 @@
+import os, dask, ray
+import pennylane as qml
+from pilot.pilot_compute_service import PilotComputeService
+from distributed import Client, wait
+import dask.bag as db
+from engine.cluster.base_executor import Executor
+from engine.cluster.dask_executor import DaskExecutor   
+
+
+def initialize_client(cluster_config):
+    pilot_compute_description_dask = cluster_config
+    pcs = PilotComputeService()
+    pilot = pcs.create_pilot(pilot_compute_description=pilot_compute_description_dask)
+    pilot.wait()
+    dask_ray_client = pilot.get_client()
+    return pilot, dask_ray_client
+
+
+class PilotQuantumExecutor(Executor):
+
+    def __init__(self, cluster_config=None):
+        super().__init__()
+        self.cluster_config = cluster_config or {}
+        self.type = self.cluster_config["config"]["type"]
+        self.pilot, self.client = initialize_client(self.cluster_config["config"])
+
+    def close(self):
+        # self.client.close()
+        self.pilot.cancel()
+        
+
+    def get_client(self):
```

Comment:
Shouldn't all the get_client, submit methods, be part of pilot-quantum framework, so we don't duplicate here?